<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorPanelUI</class>
 <widget class="QWidget" name="EditorPanelUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>166</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxHeight">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>40</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox#spinBoxHeight {
background:lightgray;
border:&quot;1px solid black&quot;;
border-radius:2px;
font-family:ArialMT;
font-size:12px;
color:#3B3B3B;
}
QSpinBox#spinBoxHeight::down-button {
image: url(:/mate/action_ico/mind/Icons8-Ios7-Arrows-Down-4.ico);
border:none;
}
QSpinBox#spinBoxHeight::up-button {
image: url(:/mate/action_ico/mind/Icons8-Ios7-Arrows-Up-4.ico);
border:none;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxWigth">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox#spinBoxWigth {
background:lightgray;
border:&quot;1px solid black&quot;;
border-radius:2px;
font-family:ArialMT;
font-size:12px;
color:#3B3B3B;
}
QSpinBox#spinBoxWigth::down-button {
border:none;
image: url(:/mate/action_ico/mind/Icons8-Ios7-Arrows-Down-4.ico);
}
QSpinBox#spinBoxWigth::up-button {
image: url(:/mate/action_ico/mind/Icons8-Ios7-Arrows-Up-4.ico);
border:none;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPesent">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>43</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox#spinBoxPesent {
background:lightgray;
border:&quot;1px solid black&quot;;
border-radius:2px;
font-family:ArialMT;
font-size:12px;
color:#3B3B3B;
}
QSpinBox#spinBoxPesent::down-button {
image: url(:/mate/action_ico/mind/Icons8-Ios7-Arrows-Down-4.ico);
border:none;
}
QSpinBox#spinBoxPesent::up-button {
image: url(:/mate/action_ico/mind/Icons8-Ios7-Arrows-Up-4.ico);
border:none;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_lock">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_lock {
	border:none;
	image: url(:/mate/action_ico/mind/Iconsmind-Outline-Unlock-2.ico);
}
QPushButton#btn_lock:checked {
	image: url(:/mate/action_ico/mind/Iconsmind-Outline-Lock.ico);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/mate/action_ico/mind/Iconsmind-Outline-Resize.ico);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/mate/action_ico/mind/favicon.ico);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
